src/vivarium_gates_child_iv_iron/model_specifications/iv_iron_child.yaml

```diff
@@ -64,12 +64,17 @@ configuration:
             - "sex"
             - "age"
         disability:
-            include: ['stunting_state', 'wasting_state']
+            include:
+                - 'wasting'
```

Comment:
Should this be wasting or wasting_state?

Reply:
'wasting_state'. I'll update.